runtime/doc/os_dos.txt

```diff
@@ -266,10 +253,10 @@ create a file is used:
 ==============================================================================
 9. Shell option default					*dos-shell*
 
-The default for the 'sh' ('shell') option is "command.com" on Windows 95 and
-"cmd.exe" on Windows NT.  If SHELL is defined, Vim uses SHELL instead, and if
-SHELL is not defined but COMSPEC is, Vim uses COMSPEC.  Vim starts external
-commands with "<shell> /c <command_name>".  Typing CTRL-Z starts a new command
+The default for the 'sh' ('shell') option is "cmd.exe" on Windows.
```

Comment:
We have the short forms later on for other ones. But maybe swap so it reads:

`'shell' ('sh')`

Or just remove short forms altogether?